### PR TITLE
Switch MC Java to lunar launcher

### PR DESCRIPTION
**Pull request by mobilegmYT**
Initial PR. Will need @gl91306 to have a look and make sure everything is good and all downloaded files are cleaned up properly. This is for the future when Lunar has full MS acc support fixed.

### CODE REVIEW

**Comment by mobilegmYT** [comment]
Also I know that @gl91306 didn't want pi-apps to have lunar added, but perhaps a compromise could be worked out?

**Comment by Botspot** [comment]
> Also I know that @gl91306 didn't want pi-apps to have lunar added, but perhaps a compromise could be worked out?

Probably so. And if not, how hard would it be to use the stock version of Lunar Client?

**Comment by mobilegmYT** [comment]
> > Also I know that @gl91306 didn't want pi-apps to have lunar added, but perhaps a compromise could be worked out?
> 
> Probably so. And if not, how hard would it be to use the stock version of Lunar Client?

It's all written in electron and javascript so we'd need box64, as well as compiling some java libraries maybe.

File: apps/Minecraft Java/install-64 [hunk 1]
```diff
@@ -7,51 +7,97 @@ function error {
...
 
-#Create desktop shortcut
-cd ~/.local/share/applications/
+#Run modprobe fuse
+sudo modprobe fuse
+
+cd ~
+
+#Download client and assorted java stuff
+wget https://github.com/gl91306/lunar/raw/master/lunarclient-2.7.3c-armv7l.AppImage
```
Comment (Jai-JAP) [inline apps/Minecraft Java/install-64:49]: This should be ` wget https://github.com/gl91306/lunar/raw/master/lunarclient-2.7.3c-aarch64.AppImage `

**Comment by mobilegmYT** [comment]
Closing since it'll be added in a different way

**Pull request closed without merging**